sympy/assumptions/assume.py

```diff
@@ -94,7 +94,8 @@ def func(self):
 
     @cacheit
     def sort_key(self, order=None):
-        return self.class_key(), (2, (self.func.name, self.arg.sort_key())), S.One.sort_key(), S.One
+        return (self.class_key(), (2, (self.func.name, self.arg.sort_key())),
+               S.One.sort_key(), S.One)
```

Comment:
E128: continuation line under-indented for visual indent.
Put one more space before `S.One.sort_key(), S.One)`